### PR TITLE
Fix compatibility of minimum PHP version check with older versions

### LINKED ISSUE #20350: PHP Fatal error:  Default value for parameters with a class type hint can only be NULL
Trying to use the Google Analytics Importer and reported in the gaimportlog.3..log file is this error:

PHP Fatal error:  Default value for parameters with a class type hint can only be NULL in /.../core/testMinimumPhpVersion.php on line 128

Running PHP 8.2

### PR DESCRIPTION
**Pull request by bx80**
### Description:

Fixes #20350 

The `testMinimumPhpVersion.php` check contains type hints which are not compatible with PHP < 7. When attempting  to run command line tasks with PHP 5.6 this caused the following error to be shown instead of an incompatible version warning:

`PHP Fatal error:  Default value for parameters with a class type hint can only be NULL in /var/www/matomo/core/testMinimumPhpVersion.php on line 128` 

Although the minimum PHP version for Matomo is 7.2+, this specific file needs to be able to run on any PHP version > 4 so it can show the version error.

This PR removes the type hints.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by sgiehl — CHANGES_REQUESTED** [review]
@bx80 The file still doesn't seem to be compatible with older PHP versions. There is a usage of a short array syntax on line 215.
Also we can't get the file compatible for PHP lower than 5.3 actually, as we are using namespaced classes inside it. 
Btw. you can simply paste the files content on https://3v4l.org/ and check which php version it would run with...

**Comment by bx80** [comment]
@sgiehl I'd just tested locally with PHP5.6, hard to believe anyone would still be running < 5.5 at this stage almost 10yrs since it was released. I've removed the short array syntax and updated the header comment to state that the file should be compatible with PHP 5.3 (since we're using namespaces). Thanks for the 3v4l link :+1:

File: core/testMinimumPhpVersion.php [hunk 1]
```diff
@@ -73,8 +73,6 @@
     }
 }
 
-define('PAGE_TITLE_WHEN_ERROR', 'Matomo &rsaquo; Error');
```
Comment (sgiehl) [inline core/testMinimumPhpVersion.php]: Any reason for removing that one. If I see that correct, it is still in use...

Comment (bx80) [reply core/testMinimumPhpVersion.php]: Fat fingers + lack of attention to detail :man_facepalming: 
Thanks for the catch, I've added it back.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**